### ISSUE TITLE
Cannot install Beam-Wallet-v.3.1.5765 on Windows 10.

### ISSUE THREAD

**Issue by Zirillian** [BeamMW/beam#743]
**Cannot install Beam-Wallet-2.1.4915-repack over 2.0.4739.exe on Windows 10.**

Since versions of the Beam Wallet starting with 2.1.XXXX, I cannot install them over the latest version of the wallet that works for me, 2.0.4739.  It gives me an error saying it can't uninstall the old version, contact technical support. The wallet was originally installed on an external SSD hard drive (no issues until 2.1); the normal files within AppData on the C drive still existed, however.

I have tried various workarounds, and none of them work. 

-uninstalling and reinstalling 2.1.X fresh
-trying to upgrade to older versions of 2.1.X than the latest
-paying for fancy registry cleaning/uninstaller pro software to try to remove any traces of the Beam Wallet on my computer, then installing fresh
- installing from different locations (default location on C, SSD again, etc. etc.)
- various permutations of the above

The wallet is currently installed at the default place on the C drive (C:/Program Files, 2.0.4739 and does otherwise work fine.

Per a suggestion from Raskul, I also edited the .cfg file to change the location it thinks AppData is in, to the various places it could be depending on where I installed the wallet. No bones.

Do I have to reformat my hard drive/reinstall Windows to get this thing to work, or are there other options?

* Windows 10 - latest (+ prior versions)
* I also have BitDefender installed; have tried multiple ways of installing from deactivating entirely to turning certain things off while installing. None work.

![Annotation 2019-06-26 212559](https://user-images.githubusercontent.com/46380317/60226814-da645d80-985a-11e9-969f-5872117760a2.jpg)
[beam_ui_19_06_26_18_53_57.log](https://github.com/BeamMW/beam/files/3332542/beam_ui_19_06_26_18_53_57.log)

**Comment by gingervik** [comment]
hi @Zirillian have you tried to delete wallet via Control Panel?

**Comment by Zirillian** [comment]
@ginervik Do you mean uninstall from the control panel, or something else? I've tried using the BEAM uninstaller and a third party uninstaller program (with the aim of trying to find all traces of the wallet on the hard drive and registry).

**Comment by Zirillian** [comment]
> hi @Zirillian have you tried to delete wallet via Control Panel?

@gingervik Do you mean uninstall from the control panel, or something else? I've tried using the BEAM uninstaller and a third party uninstaller program (with the aim of trying to find all traces of the wallet on the hard drive and registry).

**Comment by Zirillian** [comment]
Edit: The issue still exists on 2.2.5635 @gingervik

**Comment by appkoders** [comment]
I would try running process monitor and see what registry or files it was trying to access/delete. Of course this is an issue that needs to be fixed but if you really want to install it now then you could trace what files/registry it is trying to remove or what is flagging the installer to think that an older version is still installed.

**Comment by gingervik** [comment]
hi @Zirillian do you still have this issue?

**Comment by Zirillian** [comment]
> hi @Zirillian do you still have this issue?

Yes @gingervik . I last tried two days ago on v. 3.0.5660. No change...the error message looks exactly the same.

If you want to close this issue you can I guess. I've pretty much given up on getting this to work, and can no longer spend time trying to figure it out.  I also tried @appkoders solution but can't really find anything interesting by studying that screen to understand what it's trying to pull from other than the obvious.

The only solution in my mind is a system reformat which I am currently not willing to do for this.

**Comment by gingervik** [comment]
@Zirillian i still advice you to delete all beam wallets via control panal and then try to install the last version of our wallet

**Comment by Zirillian** [comment]
@gingervik There is nothing in the control panel as no Beam wallet is installed, hence the issue.  I've tried reinstalling the latest version that works (2.0.4739), then doing that, and it doesn't change the outcome.

**Comment by gingervik** [comment]
@Webonnix please, help

**Comment by Webonnix** [comment]
Hi @Zirillian
This issue is still relevant ?
---
If,yes. Try to do next:
1) win+r -> regedit -> ctrl +f -> remove mention of  Beam
2) Manually delete the Beam folder 
3) Restart PC & try to install latest version

**Comment by Zirillian** [comment]
@Webonnix I've done the steps you've listed (again) just now, and still no change in the outcome. This is now trying version 3.1.5765.

I've done similar steps previously, also.  

If you want to close this issue, so be it.  I'm frankly done trying.  The only thing that will solve this is reformatting my hard drive; I've tried everything else multiple times on multiple different wallet versions to no avail.

![Annotation 2019-08-28 223518](https://user-images.githubusercontent.com/46380317/63906038-3d51ac80-c9e4-11e9-9ef8-71b7ebf0bf0a.jpg)

**Comment by Webonnix** [comment]
Hi @Zirillian
Let's not give up. I want to help you with this issue. 
Providing an event logs (or remote access - if it possible), will help us understand and resolve this issue more quickly
However, let's do the following:
1) Remove any traces of the Beam installation by uninstalling Beam (run as admin: [uninstall](https://dropmefiles.com/GmRwg))
2) Run the [Microsoft utility](https://support.microsoft.com/en-us/help/17588/windows-fix-problems-that-block-programs-being-installed-or-removed) to repair corrupted registry keys that prevents programs from being completely uninstalled or blocking new installations and updates.
3) Reboot PC,than download and install the latest version
Thanks
-----
Also, there are other options:
You can install the wallet on another PC and copy the Beam Wallet directory to your PC and then run.
Besides,latest version without installer: [SFX archive](https://dropmefiles.com/e5Pl8)

**Comment by Zirillian** [comment]
@Webonnix your latest steps worked!
On (1) - everything was already uninstalled, so there was nothing to do.
(2) This seems to have done it. I did not have any corrupted keys or any issues found with the troubleshooter. Nevertheless, I went through and deleted any programs it found that I "had trouble uninstalling" according to the troubleshooter, that I did not want on my system.  None of them were directly related to BEAM in any way, with the exception of something like a "Name unavailable" program.  I bet the "Name unavailable" program/registry entry was it. I removed a number of others I didn't need for S&G's, rebooted, and successfully installed the latest BEAM wallet to the default directory.

Thanks for your help!  Hopefully my experience is helpful to others.  The Microsoft Utility was the fixer...despite doing countless registry repairs and deep clean uninstallers, none of them removed the wallet in its entirety (I guess). Only that utility did it.
